scripts/package_build.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+"""Build packages from a given recipe."""
+
+import argparse
+import logging
+import sys
+import docker
+from toltec.builder import Builder
+from toltec.util import argparse_add_verbose, LOGGING_FORMAT
+
+parser = argparse.ArgumentParser(description=sys.modules[__name__].__doc__)
```

Comment:
This can get shortened

```suggestion
parser = argparse.ArgumentParser(description=__doc__)
```

(If you want to have a multiline docstring that is properly displayed in the help as well, specify `formatter_class=argparse.RawDescriptionHelpFormatter` as well (otherwise newlines get ignored).)

Reply:
Oh, nice, didn’t know this could be accessed directly. Thanks!